### ISSUE TITLE
This Mod working witdh phpbb 3.3?

### ISSUE THREAD

**Issue by darkestsheep** [tas2580/phpBB-3-1-SEO-URLs#68]
see title, thanks.

**Comment by zetrader** [comment]
Yes, last version i tested, was phpbb 3.3.2, it works, but until php 7.4 (php 7.2, 7.3 or 7.4 are ok), if it's php 8.0 the extension makes some errors.
